packages/react-ui-components/src/components/SchemaForm/Widgets/CheckboxesWidget/utils.ts

```diff
@@ -0,0 +1,100 @@
+import { IRadioButton } from '@kelvininc/ui-components';
+import { ALL_BUTTON_VALUE } from './config';
+import { ICheckboxConfig } from './types';
+
+export const buildRadioButtons = <T extends { label: string; value: string }>(
```

Comment:
is there any technical limitation that forbids typing the object that T extends from?